src/Autofac.Integration.SignalR/Autofac.Integration.SignalR.csproj

```diff
@@ -80,8 +59,22 @@
     </CodeAnalysisDictionary>
   </ItemGroup>
   <ItemGroup>
+    <None Include="app.config" />
     <None Include="Autofac.Integration.SignalR.nuspec" />
-    <None Include="packages.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Autofac">
+      <Version>5.0.0</Version>
+    </PackageReference>
+    <PackageReference Include="Microsoft.AspNet.SignalR.Core">
+      <Version>2.1.1</Version>
+    </PackageReference>
+    <PackageReference Include="Microsoft.Owin.Security">
+      <Version>3.0.0</Version>
+    </PackageReference>
+    <PackageReference Include="Newtonsoft.Json">
+      <Version>5.0.8</Version>
+    </PackageReference>
```

Comment:
Do we need explicit references to these packages still? I suspect they are transitive dependencies for the Microsoft.AspNet.Signalr.Core library.

Sorry to keep adding all these changes, just want to use the opportunity to tidy things up (since we don't release this package very often).

Reply:
It's okay to remove Newtonsoft.Json (5.0.8), because Microsoft.AspNet.SignalR.Core is adding Newtonsoft.Json (6.0.4) implicitly. But if I remove Microsoft.Owin.Security (3.0.0), then Microsoft.Owin.Security (2.1.0) will get added implicitly.

Reply:
Ok, understood.